### PR TITLE
OpenSSL 1.1.1d RPMs

### LINKED ISSUE #31: TLS 1.3 final spec is implemented in OpenSSL 1.1.1+ only
Please upgrade to 1.1.1+ of OpenSSL.

### PR DESCRIPTION
**Pull request by arcivanov**
fixes #31

### CODE REVIEW

File: rpm/SPECS/openresty-asan.spec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Name:           openresty-asan
-Version:        1.15.8.2
-Release:        3%{?dist}
+Version:        1.17.1.1
```
Comment (agentzh) [inline rpm/SPECS/openresty-asan.spec]: 1.17.1.1 is not released yet. Also this change is unrelated to the current PR's title.

File: rpm/SPECS/openresty-openssl-asan.spec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Name:               openresty-openssl-asan
-Version:            1.1.0k
-Release:            4%{?dist}
+Version:            1.1.1d
```
Comment (agentzh) [inline rpm/SPECS/openresty-openssl-asan.spec]: Because openssl may break API compatibility across major release version numbers, we'd better introduce different package names like `openresty-openssl111` and `openresty-openssl111-asan` to make sure older openresty package versions still work.

File: rpm/SPECS/openresty-asan.spec (hunk 1 above)
Comment (arcivanov) [reply rpm/SPECS/openresty-asan.spec]: Very well, I'll fork it out.

File: rpm/SPECS/openresty-openssl-asan.spec (hunk 2 above)
Comment (arcivanov) [reply rpm/SPECS/openresty-openssl-asan.spec]: Very well, I'll make the change.

**Comment by arcivanov** [comment]
Changed as requested @agentzh

**Comment by arcivanov** [comment]
@agentzh Anything else needs fixing?
